tests/tasks_test.php

```diff
@@ -61,6 +61,7 @@ public function test_run_scheduled_tasks() {
             'push_objects_to_storage',
             'recover_error_objects',
             'check_objects_location',
+            'sync_filedir_location',
```

Comment:
This new task is broadly the same as the check_objects_location, but one is looking for orphaned db records, and the other is looking for orphaned files. I'm wondering wether we should name them similarly so this relationship is more obvious.

eg check_objects_location vs check_filedir_location

or even renaming them both:

check_location_db and check_location_local_filedir

Reply:
renamed to check_filedir_location